### PR TITLE
Update funding, approval, accountability

### PR DESCRIPTION
**Pull request by ttriche**
**Did you add yourself as a [contributor](https://github.com/Benjamin-Lee/deep-rules/blob/master/contributors.md) if this is your first contribution?**

- [ x ] Yes, I added myself or am already a contributor

**Any more details?**
Update accountability, affiliations, approval, and funding sections.

### CODE REVIEW

**Review by agitter — CHANGES_REQUESTED** [review]
Thanks for the updates @ttriche.  Can you add the `postcode` field too please?

**Comment by ttriche** [comment]
Should be “49503” — my laptop is disassembled at the moment (coffee spill) and editing the file on my phone has so far failed 

--t

File: content/metadata.yaml [hunk 1]
```diff
@@ -210,10 +210,19 @@ authors:
...
+      - Department of Pediatrics, College of Human Medicine, Michigan State University
       - Department of Translational Genomics, Keck School of Medicine, University of Southern California
+    conflicts: None
+    funders: 
+      - National Institute of Allergy and Infectious Diseases (R21AI153997)
+      - Michelle Lunn Hope Foundation
+      - Folz Fund for Cancer Research
+      - Grand Rapids Community Foundation
+    approval: yes
+    accountable: yes
```
Comment (agitter) [inline content/metadata.yaml:225]:
```suggestion
    accountable: yes
    postcode: "49503"
```

Thanks, I added it.

**Review by agitter — APPROVED** [review]

**Comment by ttriche** [comment]
Thank you!

--t

**Review by cgreene — APPROVED** [review]

**Pull request merged**